tests/FakeItEasy.Specs/DummyCreationSpecs.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using Xbehave;
+
+    public abstract class DummyCreationSpecsBase

```

Comment:
There were no specs for dummy creation, so I added some. Probably partially redundant with `DummyTests`